syntax="proto3";

package Helloworld;

service HelloworldService{
    rpc GetReply(MsgReq) returns (MsgResp){}

    rpc GetStreamReply(stream MsgReq) returns (stream MsgResp){}
}

message MsgReq{
    string name = 1;
    string ip = 2;
}

message MsgResp{
    string reply = 1;
}